kernel/init.c

```diff
@@ -156,7 +158,7 @@ void _bss_zero(void)
 #endif
 }
 
-
+      
 #ifdef CONFIG_XIP
```

Comment:
CHECKPATCH ERROR:
 open brace '{' following function declarations go on the next line